README.md

```diff
@@ -1 +1,47 @@
 # mint-signup-page
+
+> It is a sign up form prototype for Intuit website created for educational purposes.
+
+![screenshot](./images/screenshot.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- Visual Studio Code
+
+## Live Demo
+
+[Live Demo Link](https://github.com/Jarfsoft/mint-signup-page)
```

Comment:
The link is not working. Please check and provide a working live demo link. [This website](https://raw.githack.com/) might help.